src/lib/capacityEngine.js

```diff
@@ -114,10 +199,12 @@ function capacityEngine(store, productIds, time, period = 30) {
         productId: Number(productId),
         availableCapacity: capacityInAssets,
         usedCapacity: capacityUsedNXM,
+        utilizationRate: getUtilizationRate(capacityInAssets, capacityUsedNXM),
```

Comment:
nit: here we can just pass `capacityAvailableNXM` instead of `capacityInAssets` and we can skip filtering the array